senders/example_direct_test.go

```diff
@@ -0,0 +1,14 @@
+package senders_test
+
+import wavefront "github.com/wavefronthq/wavefront-sdk-go/senders"
+
+func ExampleDirect() {
```

Comment:
GO is really picky about how you name your examples.  Try renaming this ExampleNewSender_direct().  This name tells go to associate this example with the NewSender function.  GO lets you add underscore and a lower case identifier in case you have multiple examples for the same function.  In your case ExampleNewSender_direct() and ExampleNewSender_proxy() would both be examples for the NewSender function.